sql/storm-sql-core/pom.xml

```diff
@@ -66,10 +66,34 @@
             <artifactId>calcite-core</artifactId>
             <version>${calcite.version}</version>
             <exclusions>
+                <exclusion>
```

Comment:
They're for supporting another parts of Calcite and unneeded dependencies for us. Some dependencies are something already we have, so also good to avoid the version conflict.

Reply:
Makes sense if we don't need them.

If you're trying to avoid version conflicts for transitive dependencies I think adding a `<dependencyManagement>` section to set the versions is better. It lets you set which version of the dependency you want, and then all transitive includes of that dependency will be the specified version. I think we do this with e.g. the Jackson dependencies in the Storm parent pom.